test/models/post_test.rb

```diff
@@ -1,17 +1,31 @@
 require 'test_helper'
 
 class PostTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
 
   test "Add Post" do
+    post = Post.new({
+      title: "Hello World",
+      body:  "Hello KangKyu, This is the world."
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.